reflex-algos/src/main/python/mlops/parallelm/mlops/time_capture/plot_functions.py

```diff
@@ -181,8 +214,48 @@ def print_matrix(self):
                         print(tabulate(matrix_df, headers='keys', tablefmt='psql'))
                     else:
                         print(matrix_df)
-            except Exception as err:
+            except Exception:
+                pass
+
+    def get_matrix_df(self, name):
+        """
+        getting the DF of a matrix according to the name:
+
+        :param self:
+        :return: matrix_df
+        """
+        matrix_df = pd.DataFrame()
+        for filename in self._file_names:
```

Comment:
add comments - i dont really follow most of it

Reply:
ok